### PR TITLE
Sign Out & Dropdown Menu

### PR DESCRIPTION
**Pull request by horsemaker**
- Added Sign Out feature
- Added Dropdown Menu
image

### CODE REVIEW

File: src/frontend/hooks/useOnClickOutside.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { useEffect } from "react";
+
+export const useOnClickOutside = (ref, handler) => {
+  useEffect(() => {
+    const listener = (event) => {
+      if (ref.current && !ref.current.contains(event.target)) {
+        handler();
+      }
```
Comment (t007rushi) [inline src/frontend/hooks/useOnClickOutside.js:8]: Liked the conditional rendering

**Review by t007rushi — APPROVED** [review]
Drop Down works Look for its responsiveness

File: src/frontend/components/AccountDropdownMenu/AccountDropdownMenu.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    setAuth({
+      status: false,
+      token: null,
+      user: null,
+    });
+  };
+
+  return (
+    <section
+      ref={dropdownMenuRef}
```
Comment (vijaytembugade) [inline src/frontend/components/AccountDropdownMenu/AccountDropdownMenu.js:30]: This Thing is Nice. Hope you can teach us `useRef`

**Review by vijaytembugade — APPROVED** [review]
Dropdown Looks good. Good work

image

**Pull request merged**